src/Command/SequentialProcessQueueCommand.php

```diff
@@ -90,6 +90,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
                 // call the garbage collector to avoid too many connections & memory issue
                 if (($i + 1) % 200 === 0) {
+                    // Clear runtime cache so that pimcore memory usage does not go overboard 🫠
+                    // https://github.com/pimcore/pimcore/issues/15866
+                    // https://docs.pimcore.com/platform/Pimcore/Objects/External_System_Interaction/#memory-issues
+                    \Pimcore\RuntimeCache::clear();
```

Comment:
```suggestion
                    \Pimcore\Cache\RuntimeCache::clear();
```
Thank you for your PR! 
Have you tried? it shouldn't be working as  the namespace is wrong, right?
weird, afaik collectGarbage should have done the magic 
https://github.com/pimcore/pimcore/blob/2593a4e9c2f99812d6cdb726cb6834dd728576ab/lib/Pimcore.php#L160-L164


I'd like to suggest to remove this comment there, it would easily found by a git blame and the function is self-explanatory already.

Reply:
Yeah you are right when forking it I did a mistake - but I can remove the comments 👍